cmd/build.go

```diff
@@ -47,11 +46,13 @@ var buildCmd = &cobra.Command{
 func init() {
 	Promu.AddCommand(buildCmd)
 
+	buildCmd.Flags().Bool("cgo", false, "Enable CGO")
```

Comment:
I think moving go specific attributes in a go namespace is a good idea. The earlier we do that, the less painful it'll be. We can take care of the repos which need updating.